elasticdl_preprocessing/layers/discretization.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import, division, print_function
+
+import tensorflow as tf
+from tensorflow.python.ops import math_ops
+
+
+class Discretization(tf.keras.layers.Layer):
+    """Buckets data into discrete ranges.
+
+    TensorFlow 2.2 has developed `tf.keras.layers.preprocessing.Discretization`
+    but not released it yet. So the layer is a simple temporary version.
+    https://github.com/tensorflow/tensorflow/blob/r2.2/tensorflow/python/keras/layers/preprocessing/discretization.py
```

Comment:
Cannot you use it via `tensorflow.python.xxxx`?

Reply:
Good suggestion!